src/Admin/Service/Settings.php

```diff
@@ -0,0 +1,298 @@
+<?php
+/**
```

Comment:
Shouldn't this script be called SettingsService.php ?

Reply:
Doesn't have to be